### ISSUE TITLE
The Long Dark (305620)

### ISSUE THREAD

**Issue by stratvox** [ValveSoftware/Proton#3892]
# Compatibility Report
- Name of the game with compatibility issues: The Long Dark
- Steam AppID of the game: 305620

## System Information
- GPU: radeon rx 5700xt
- Driver/LLVM version: 20.2.0-devel (git-989619c 2020-05-20 eoan-oibaf-ppa)
[steam-305620.log](https://github.com/ValveSoftware/Proton/files/4656854/steam-305620.log)

- Kernel version: 5.6.14-050614-generic
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/stratvox/83e0b1fcbf414cc427ce683e7bf5f681
- Proton version: 5.0-7

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 

The game will not load the intro movie, rendering only a black screen. Even if the intro movie is escaped out of it will still not render anything. 

It looks like the relevant line from output_log.txt is:

`WindowsVideoMedia error 0x80004001 while reading Z:\home\jack\.local\share\Steam\steamapps\common\TheLongDark\tld_Data\StreamingAssets\Video\Windows\intro_cards.mp4`

## Reproduction

Select "force use of a steam compatibility tool", select proton 5.0-7, and install the game.

Note that the game worked flawlessly in the steam proton environment before the latest update to the game.

**Comment by kisak-valve** [comment]
Hello @stratvox, looks like this game would benefit from support for Media Foundation maturing.

**Comment by stratvox** [comment]
Aye, I figured that from reading other bug reports that mentioned that specific error code (WindowsVideoMedia error 0x80004001). Thought I'd pile on by mentioning it wrt this game as well.

Any ideas as to when we might expect to see that happen?

ETA: alternatively, is there another way it could be possible to jam support into Wine or some such so that I can get this game working again?

**Comment by kisak-valve** [comment]
For this game, the easiest answer is to use the native linux version. There's [ongoing work](https://source.winehq.org/git/wine.git/?a=search&h=HEAD&st=commit&s=mf) upstream to support Media Foundation which will eventually make its way into Proton.

**Comment by stratvox** [comment]
Yeah, unfortunately the native linux version has an issue that results in a ridiculous amount of landscape flickering caused by z-order fighting; it really has to be seen to be believed.

Okay, I guess I'll keep an eye on it, as well as leaning on the developer to fix the linux-native version's issues (which is to say to lean on their vendor to fix them in the engine they're using).

Thanks for your time on this.

**Comment by Leopard1907** [comment]
@stratvox Maybe it is not the same issue but you should give it a try if you didn't already.

https://steamcommunity.com/app/305620/discussions/0/1741103267270894219/

**Comment by KarloKick** [comment]
> For this game, the easiest answer is to use the native linux version. There's [ongoing work](https://source.winehq.org/git/wine.git/?a=search&h=HEAD&st=commit&s=mf) upstream to support Media Foundation which will eventually make its way into Proton.

The native Linux Version is also broken. The game will simply not start anymore. This made me switch to Proton in the first place. Too bad - worked really well with Proton before the update.

**Comment by arrowgent** [comment]
hmm.... i saw this and wondered if they broke the game somehow
the usual 30-40 second launch wait is still there
(native linux)
but i started the game fine and joined a challenge to look for graphics glitches
nothing unusual

oh.  well, ive never played the story mode.  that might be quite different.

and if you need MF support there are options
i believe protontricks has an option to install the basic MF

a bit off topic since this reply isnt targeted for proton.
maybe find out why this game is causing graphics glitches for you in native linux
because my game runs flawless.

nvidia 1070 v440.82
linuxmint 19.3/ubuntu 18.04LTS

**Comment by kisak-valve** [comment]
The Long Dark (305620)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/5713.
@jchiversAND posted on 2022-03-23T18:59:56:

# Compatibility Report
- Name of the game with compatibility issues: The Long Dark
- Steam AppID of the game: 305620

## System Information
Steam Deck 512GB
- Proton version: 7.0-1, Experimental, 7-9

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
Framerate halves instantly with any camera movement.

Even with minimum graphics settings, 480p resolution, staring directly at the floor, the second you move with either the right or left stick, framerate cuts in half from 60fps to around 35fps.

As soon as you stop moving, framerate returns to 60.

Plenty of performance headroom when not moving, performance bug linked to movement/camera movement.

Unsure how this passed verification as this is a Steam Deck Verified game.

## Reproduction
Stand still, check fps
Move character or camera, framerate drops almost half

**Comment by kisak-valve** [comment]
The Long Dark (305620)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/8204.
@PavelDobCZ23 posted on 2024-10-31T10:23:48:

# Compatibility Report
- Name of the game with compatibility issues: **The Long Dark**
- Steam AppID of the game: **305620**

## System Information
- GPU: NVIDIA GeForce RTX 3060 Laptop GPU
- Video driver version: 4.6.0 NVIDIA 560.35.03
- Kernel version: 6.11.5-300.fc41.x86_64
- Link to full system information report as [Gist](https://gist.github.com/PavelDobCZ23/4f11ab16ba0db2158d2a9d3c1addcdf3)
- Proton version: **9.0-3**

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms
After loading the game past all the disclaimers, it begins to periodically **freeze every second**. It is very noticeable and makes the game very uncomfortable to play. There seems to be no configuration, where this can be solved (like changing game's settings or adding a proton environment variable). The issue is present on versions of Proton 8 and above. The oldest version the game can run on is Proton 7, where this issue isn't present. It might be related to an error `Failed to find library for L"Windows.System.DispatcherQueue"`, which can be found in the attached log and seems to be printed out often as well.
It can be demonstrated with this graph generated by [MangoHud](https://github.com/flightlessmango/MangoHud) (NOTE: the problem also occurs when not using it).
![image](https://github.com/user-attachments/assets/42d95e85-d7ea-468b-ac31-28f095d89559)

## Reproduction
1. Go to game's "Properties" >"Compatibility" > "Force use of specific Steam Play compatibility tool" = "Proton 9.0-3"
2. Open the game, wait for it to load, create a new sandbox save file and play.
3. Notice periodic freezes every second affecting the graphical output. (Easy to notice when moving around.)

## Log
[steam-305620.log](https://github.com/user-attachments/files/17586138/steam-305620.log)

**Comment by luisalvarado** [comment]
Currently using Proton Experimental on Kubuntu 25.10. When I use the proton version, the game has stuttering every once in a while and keys get stuck. For example if I press the A & W to move sideways, one of them gets stuck. The mouse is also extra sensitive and trying to get a single headshot with a bow or rifle is a nightmare compared to using the native version of the game.

I also then tested with Proton 10 (the latest from today) and it had the same issue. Did not test with Proton 11 Beta. Basically moving around using proton feels odd compared to native which is much more smooth. I did not see a performance issue and using MangoHud it did not show any performance degradation, just input issues.
